[INFO -- 2018-09-15 23:55:47]
Starting flip_value script
version: 1.1
[INFO -- 2018-09-15 23:55:47]
Initialization strings:
[INFO -- 2018-09-15 23:55:47]
(1) file /tmp/lreg/lreg
[INFO -- 2018-09-15 23:55:47]
(2) set args /tmp/lreg/randInput /tmp/lreg/output
[INFO -- 2018-09-15 23:55:47]
initSignal:0
[INFO -- 2018-09-15 23:55:47]
endSignal:2
[INFO -- 2018-09-15 23:55:47]
faultModel:1
[EXCEPTION -- 2018-09-15 23:55:48]
gdbException: Attempt to dereference a generic pointer.
[EXCEPTION -- 2018-09-15 23:55:49]
pythonException: Unable to exit loop in struct fields; Exiting wihtout making a bit flip
[EXCEPTION -- 2018-09-15 23:55:50]
pythonException: empty range for randrange() (0,0, 0)
[INFO -- 2018-09-15 23:55:54]
Backtrace BEGIN:
#0  __GI_raise (sig=sig@entry=6) at ../sysdeps/unix/sysv/linux/raise.c:51
#1  0x00007ffff6ec1801 in __GI_abort () at abort.c:79
#2  0x00007ffff6f0a897 in __libc_message (action=action@entry=do_abort, fmt=fmt@entry=0x7ffff7037b9a "%s\n") at ../sysdeps/posix/libc_fatal.c:181
#3  0x00007ffff6f1190a in malloc_printerr (str=str@entry=0x7ffff7039870 "double free or corruption (out)") at malloc.c:5350
#4  0x00007ffff6f18e75 in _int_free (have_lock=0, p=0x555555771790, av=0x7ffff726cc40 <main_arena>) at malloc.c:4278
#5  __GI___libc_free (mem=0x5555557717a0) at malloc.c:3124
#6  0x000055555555772a in __gnu_cxx::new_allocator<double>::deallocate (this=0x55555575a140 <x>, __p=0x5555557717a0) at /usr/include/c++/7/ext/new_allocator.h:125
#7  0x00005555555571f0 in std::allocator_traits<std::allocator<double> >::deallocate (__a=..., __p=0x5555557717a0, __n=56) at /usr/include/c++/7/bits/alloc_traits.h:462
#8  0x0000555555556944 in std::_Vector_base<double, std::allocator<double> >::_M_deallocate (this=0x55555575a140 <x>, __p=0x5555557717a0, __n=56) at /usr/include/c++/7/bits/stl_vector.h:180
#9  0x00005555555565db in std::_Vector_base<double, std::allocator<double> >::~_Vector_base (this=0x55555575a140 <x>, __in_chrg=<optimized out>) at /usr/include/c++/7/bits/stl_vector.h:162
#10 0x000055555555602d in std::vector<double, std::allocator<double> >::~vector (this=0x55555575a140 <x>, __in_chrg=<optimized out>) at /usr/include/c++/7/bits/stl_vector.h:435
#11 0x00007ffff6ec4041 in __run_exit_handlers (status=0, listp=0x7ffff726c718 <__exit_funcs>, run_list_atexit=run_list_atexit@entry=true, run_dtors=run_dtors@entry=true) at exit.c:108
#12 0x00007ffff6ec413a in __GI_exit (status=<optimized out>) at exit.c:139
#13 0x00007ffff6ea2b9e in __libc_start_main (main=0x555555555975 <main(int, char**)>, argc=3, argv=0x7fffffffde68, init=<optimized out>, fini=<optimized out>, rtld_fini=<optimized out>, stack_end=0x7fffffffde58) at ../csu/libc-start.c:344
#14 0x000055555555501a in _start ()

46	../sysdeps/unix/sysv/linux/raise.c: No such file or directory.

Backtrace END
Memory content before bitflip:0x010x000x000x00
Fault Model: Double bit-flip
base address to flip value: 0x7fffffffdb48
address max offset: 4
Memory content after  bitflip:0x010x480x000x00
frame name: __libc_message
symbol name: action
symbol filename: ../sysdeps/posix/libc_fatal.c
symbol line: 67
value: (do_abort | unknown: 18432)
value address: 0x7fffffffdb48
Type: The type is an enum.
Type sizeof: 4
Field name: do_message

[INFO -- 2018-09-15 23:55:54]
Fault Injection Successful after 7.442897319793701s
[INFO -- 2018-09-15 23:55:54]
event type: exit
[EXCEPTION -- 2018-09-15 23:55:54]
exit code: no exit code available
